### ISSUE TITLE
what's the config when you train this model?

### ISSUE THREAD

**Issue by EEWenbinWu** [HugoTini/DeepBump#1]
Hi, HugoTini! Your perfect work helps me a lot and I want to train my own data, could you please tell me the detail of training the Unet

**Comment by HugoTini** [comment]
Hi, sorry for the late answer, glad to hear this project was helpful to you :) !

As mentioned in the readme, currently, the training pipeline is not published. It might happen in the future, but I don't have plans nor time to make it public right now.

**Comment by vinmisra** [comment]
Hey @HugoTini great stuff! No need for a training code release, but I am super curious if you have any quick insights about how you've gone about it, particularly in terms of the training data preparation or loss function?

I've attempted training various types of image-to-image models from blender renders of ambientCG materials (input) to ambientCG normal maps (ground truth output), but deepbump's inferences are consistently more dramatic, even moreso than the ground truth. E.g. I've noticed it rarely outputs a flat normal map.

I'm very curious how you curated the training set / objective to make this happen!

Thanks again for sharing this beautiful work.
